python/lsst/dm/OCPS/ocps_csc.py

```diff
@@ -369,5 +318,10 @@ def get_config_pkg():
     async def configure(self, config: types.SimpleNamespace):
         self.log.info(f"Configuring with {config}")
         self.config = config
+        if self.index != OcpsIndex[self.config.instance]:
+            raise salobj.ExpectedError(
+                f"Configuration instance '{self.config.instance}'"
+                f" does not match CSC index '{self.index}'"
```

Comment:
I am pretty sure this will print a message such as "Configuration instance LATISS does not match CSC index 2", which is hard to understand without looking at the definition of the `OcpsIndex` class.

Please improve it to print the name of the current index (or the name and numeric value, if you prefer).

Reply:
Actually, `str(OcpsIndex(2))` is `OcpsIndex.LSSTComCam`.  I can make it `repr`, which is `<OcpsIndex.LSSTComCam: 2>`.

Reply:
Hm...I tried f"{enum_value}" and it printed the integer.

Reply:
I guess that's why there's `!s`.  Thanks for catching this.  I'm already changing to `!r`.